src/clj/rems/db/core.clj

```diff
@@ -8,9 +8,17 @@
             [mount.core :refer [defstate] :as mount]
             [rems.config :refer [env]]))
 
+;; See docs/architecture/010-transactions.md
+(def ^:private +hikaricp-settings+
+  {:connection-init-sql "SET lock_timeout TO '10s'; SET idle_in_transaction_session_timeout TO '20s';"})
+
 (defstate ^:dynamic *db*
-  :start (try (let [db (cond (:test (mount/args)) (conman/connect! {:jdbc-url (:test-database-url env)})
-                             (:database-url env) (conman/connect! {:jdbc-url (:database-url env)})
+  :start (try (let [db (cond (:test (mount/args)) (conman/connect! (merge +hikaricp-settings+
+                                                                          {:jdbc-url (:test-database-url env)}))
+                             (:database-url env) (conman/connect! (merge +hikaricp-settings+
+                                                                         {:jdbc-url (:database-url env)}))
+                             ;; the jndi codepath doesn't use hikari, so we can't use +hikaricp-settings+
```

Comment:
Where do we use this one?

Reply:
In tomcat installations tomcat provides the db connection for us

... but even with tomcat installations we could switch to using `:database-url`